cmd/build.go

```diff
@@ -469,6 +469,7 @@ func buildEnv(targetOS string, engineCachePath string) []string {
 	env := []string{
 		"GO111MODULE=on",
 		"CGO_LDFLAGS=" + cgoLdflags,
+		"CGO_CFLAGS=-mmacosx-version-min=10.12",
```

Comment:
Im not sure if that causes problems on other platforms. Can you make a switch for that just like the ldflags?